specs/WindowControlsOverlayConfiguration.md

```diff
@@ -0,0 +1,156 @@
+
+    wil::com_ptr<ICoreWebView2> coreWebView2_28;
+    CHECK_FAILURE(coreWebView2->QueryInterface(&coreWebView2_28));
+
+    wil::com_ptr<ICoreWebView2WindowControlsOverlaySettings> windowControlsOverlaySettings;
+    CHECK_FAILURE(coreWebView2_28->get_WindowControlsOverlaySettings(&wco_config));
+
+    CHECK_FAILURE(wco_config->put_IsEnabled(true));
+    COREWEBVIEW2_COLOR color {1, 0, 0, 225};
+    CHECK_FAILURE(wco_config->put_TitleBarBackgroundColor(color));
+}
+```
+## .NET C#
+```c#
+// WebView2 control is defined in the xaml
+// <wv2:WebView2 x:Name="webView" Source="https://www.microsoft.com/"/>
+public MainWindow() 
+{
+    InitializeComponent();
+    m_AppWindow.TitleBar.ExtendsContentIntoTitleBar = true;
+
+    CoreWebView2WindowControlsOverlaySettings config = Webview2.CoreWebivew2.WindowControlsOverlaySettings;
+    config.IsEnabled = true; 
+    config.color = Color.FromARGB(0, 0, 255);
+}
+```
+
+# API Details
+## Win32 C++
+```cpp
+/// Controller API used to configure the window controls overlay.
+/// To provide your app users with the best experience, it is important to handle webview 
+/// initialization errors appropriately. Provide your users with a way to close the window
+/// or restart the App.
+[uuid(101e36ca-7f75-5105-b9be-fea2ba61a2fd), object, pointer_default(unique)]
+interface ICoreWebView2_28 : IUnknown {
+  /// Gets the `WindowControlsOverlaySettings` object.
+  [propget] HRESULT WindowControlsOverlaySettings([out, retval] ICoreWebView2WindowControlsOverlaySettings** value);
+}
+
+/// This is the ICoreWebView2WindowControlsOverlaySettings
+[uuid(c9f7378b-8dbb-5445-bacb-08a3fdf032f0), object, pointer_default(unique)]
+interface ICoreWebView2WindowControlsOverlaySettings : IUnknown {
+    /// Gets the `Height` property.
+  [propget] HRESULT Height([out, retval] UINT32* value);
+
+
+  /// The `Height` property in raw screen pixels, allows you to set the height of the overlay and
+  /// title bar area. Defaults to 48px. There is no minimum height restriction for this API,
+  /// so it is up to the developer to make sure that the height of your window controls overlay
+  /// is enough that users can see and interact with it. We recommend using GetSystemMetrics(SM_CYCAPTION)
+  // as you minimum height.
+  /// 
+  [propput] HRESULT Height([in] UINT32 value);
+
+
+  /// Gets the `IsEnabled` property.
+  [propget] HRESULT IsEnabled([out, retval] BOOL* value);
+
+
+  /// The `IsEnabled` property allows you to opt in to using
+  /// the WebView2 window controls overlay. Defaults to `FALSE`.
+  /// 
+  /// When this property is `TRUE`, WebView2 will draw its own minimize, maximize,
+  /// and close buttons on the top right corner of the Webview2. 
+  /// 
+  /// When using this you should configure your app window to not display its default
+  /// window control buttons. You are responsible for creating a title bar for your app
+  /// by using the available space to the left of the controls overlay. In doing so, 
```

Comment:
Is this right or left conditional on the reading direction of the UI root?

Reply:
Please document what happens for RTL locales - does the controls overlay render on the left instead of right?

In the text here say 'before' instead of 'left'

Reply:
In RTL locales, the caption buttons still appear on the right side of the window, just like in LTR. The ordering of the buttons themselves is mirrored (Close | Maximize | Minimize), but their position remains on the right side of the window.

keeping that in mind 'left' actually fits correctly in the statement? please let me know if you still want it to be 'before' ?